### PR TITLE
Add resource requirements

### PR DESCRIPTION
**Pull request by anishasthana**

### CODE REVIEW

File: monitoring/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -72,7 +72,13 @@ spec:
         ports:
         - containerPort: 9090
           name: http
-        resources: {}
+        resources:
+          limits:
+            cpu: 2000m
```
Comment (maulikjs) [inline monitoring/prometheus/prometheus.yaml:77]: can you change this to 2 instead of 2000m?

File: monitoring/prometheus/prometheus.yaml [hunk 2]
```diff
@@ -72,7 +72,13 @@ spec:
...
+            memory: 2Gi
+          requests:
+            cpu: 1000m
```
Comment (maulikjs) [inline monitoring/prometheus/prometheus.yaml:80]: can you change this to 1 instead of 1000m?

**Review by harshad16 — COMMENTED** [review]
/lgtm

if the cluster that are provided to the users, has these resource then it is cool :+1:

**Comment by crobby** [comment]
1 whole cpu just for prometheus monitoring seems high....Is that approximately what it actually needs?

**Comment by crobby** [comment]
In my instance, which isn't doing much, prometheus appears to use about 3.3G of memory.  We may require a higher limit.
Screenshot from 2021-05-25 06-24-43

**Review by maulikjs — APPROVED** [review]

**Pull request merged**